### PR TITLE
feat: People page enhancement - MEED-1838 - Meeds-io/meeds#746

### PR DESCRIPTION
**Pull request by SaraBoutej**
Fix Feedback

### CODE REVIEW

File: webapp/portlet/src/main/webapp/vue-apps/people-list/components/PeopleToolbar.vue [hunk 1]
```diff
@@ -103,6 +108,9 @@ export default {
     isMobile() {
       return this.$vuetify.breakpoint.width < 768;
     },
+    isStickyMenu() {
+      return this.stickyMenu && this.$vuetify.breakpoint.width > 768;
+    }
```
Comment (boubaker) [inline webapp/portlet/src/main/webapp/vue-apps/people-list/components/PeopleToolbar.vue]: Not used

```suggestion
```

File: webapp/portlet/src/main/webapp/vue-apps/people-list/components/PeopleToolbar.vue [hunk 2]
```diff
@@ -79,9 +82,11 @@ export default {
     typing: false,
     advancedFilterCount: 0,
     mobileFilter: false,
-    iconWidth: '24px'
+    iconWidth: '24px',
+    stickyMenu: false
   }),
   created() {
+    this.stickyMenu = eXo.env.portal.stickyMenu;
```
Comment (boubaker) [inline webapp/portlet/src/main/webapp/vue-apps/people-list/components/PeopleToolbar.vue]: Not used

```suggestion
```

File: webapp/portlet/src/main/webapp/vue-apps/people-list/components/PeopleToolbar.vue [hunk 3]
```diff
@@ -79,9 +82,11 @@ export default {
     typing: false,
     advancedFilterCount: 0,
     mobileFilter: false,
-    iconWidth: '24px'
+    iconWidth: '24px',
+    stickyMenu: false
```
Comment (boubaker) [inline webapp/portlet/src/main/webapp/vue-apps/people-list/components/PeopleToolbar.vue]: Not used
```suggestion
```

Comment (SaraBoutej) [reply webapp/portlet/src/main/webapp/vue-apps/people-list/components/PeopleToolbar.vue]: removed!

File: webapp/portlet/src/main/webapp/vue-apps/people-list/components/PeopleToolbar.vue (hunk 2 above)
Comment (SaraBoutej) [reply webapp/portlet/src/main/webapp/vue-apps/people-list/components/PeopleToolbar.vue]: removed!

File: webapp/portlet/src/main/webapp/vue-apps/people-list/components/PeopleToolbar.vue (hunk 1 above)
Comment (SaraBoutej) [reply webapp/portlet/src/main/webapp/vue-apps/people-list/components/PeopleToolbar.vue]: removed!

**Review by boubaker — APPROVED** [review]

**Pull request merged**